code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -175,7 +181,7 @@
 	gain_text = "I was able to combine my greed and desires to summon an eldritch beast i have not seen before. Ever shapeshifting mass of flesh, it well knew my goals."
 	desc = "You can now summon a Stalker using a knife, a flower, a pen and a piece of paper. Stalkers can shapeshift into harmeless animals and get close to the victim."
 	cost = 1
-	required_atoms = list(/obj/item/kitchen/knife,/obj/item/reagent_containers/food/snacks/grown/poppy,/obj/item/pen,/obj/item/paper)
+	required_atoms = list(/obj/item/kitchen/knife,/obj/item/pen,/obj/item/paper)
```

Comment:
yooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

I hate to say it, but if you are making stalkers easier to make, you might want to remove the secbot form from their list of possible shapeshift forms. Being able to spam out an army of 10 sapient beepskies after raiding the curator's vending machiine might be a bit TOO strong, but then again, they only have 25 health each, so 🤷 , maybe it's fine.